PortalSwift/Sources/PortalSwift/Connect/PortalConnect.swift

```diff
@@ -198,7 +237,7 @@ public class PortalConnect: EventBus {
           id: connectData.id,
           topic: connectData.topic,
           address: self.address!,
-          chainId: String(self.chainId),
+          chainId: String(self.chainId ?? 11_155_111),
```

Comment:
It probably makes sense to error if `self.chainId` is nil instead of defaulting to Sepolia